docs/source/dataframe_models.md

```diff
@@ -11,20 +11,8 @@ file_format: mystnb
 
 # DataFrame Models
 
-Formerly known as `SchemaModel`.
```

Comment:
this has been removed in a different PR. can we restrict changes in this PR to ibis-only changes?